.eslintrc.json

```diff
@@ -10,7 +10,9 @@
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-this-alias": "off",
         "@typescript-eslint/no-empty-function": "off",
-        "@typescript-eslint/no-empty-interface": "off"
+        "@typescript-eslint/no-empty-interface": "off",
+        "@typescript-eslint/no-inferrable-types": "off",
+        "@typescript-eslint/ban-types": "off"
```

Comment:
Similar question about `ban-types`, though I think this rule is much more important. It can catch a lot of bugs that aren't so obvious.